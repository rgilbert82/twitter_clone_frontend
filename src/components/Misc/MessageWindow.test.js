import React from 'react';
import { shallow } from 'enzyme';
import { MessageWindow } from '.';

describe(MessageWindow, () => {
  let component;

  beforeEach(() => {
    component = <MessageWindow message='Error Message' closeMessage={jest.fn()} />;
  });

  it("renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
