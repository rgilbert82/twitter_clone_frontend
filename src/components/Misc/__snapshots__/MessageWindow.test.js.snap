// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class MessageWindow extends _react2.default.Component {
  constructor(props) {
    super(props);
    this.closeMessage = this.closeMessage.bind(this);
  }

  closeMessage(e) {
    e.preventDefault();
    this.props.closeMessage();
  }

  render() {
    return (
      _react2.default.createElement(_reactBootstrap.Alert, { className: 'message_window', __source: { fileName: _jsxFileName, lineNumber: 17 }, __self: this },
        _react2.default.createElement('p', { __source: { fileName: _jsxFileName, lineNumber: 18 }, __self: this }, this.props.message),
        _react2.default.createElement('a', { onClick: this.closeMessage, href: '', __source: { fileName: _jsxFileName, lineNumber: 19 }, __self: this }, 'close')));


  }} renders properly 1`] = `
<MessageWindow
  closeMessage={[Function]}
  message="Error Message"
/>
`;
